src/test/java/org/kcctl/command/ApplyCommandTest.java

```diff
@@ -140,4 +140,14 @@ public void fail_on_first_error() {
         assertThat(context.output().toString()).doesNotContain("Created connector");
     }
 
+    @Test
+    public void should_gracefully_handle_missing_connector_class() {
+        var path = Paths.get("src", "test", "resources", "nonexistent.json");
+
+        int exitCode = context.commandLine().execute("-f", path.toAbsolutePath().toString());
+
+        assertThat(exitCode).isEqualTo(CommandLine.ExitCode.SOFTWARE);
+        assertThat(context.output().toString()).contains("Specified class isn't a valid connector type");
```

Comment:
I wanted to add an assertion about it actually showing the list of available connectors, but was having some trouble running the integration tests locally (containers were hanging on startup).

While less powerful, this test still guards against a regression that would re-introduce the NPE addressed by this PR.

Reply:
Is that container issue on Mac M1? If so, I've recently pushed an update which seems to help for the Kafka image, but Connect still seems slow. It's not quite clear to my why, as the Debezium 2.0 images have an Arm flaviour. Perhaps Testcontainers is not pulling that in for some reason?

Reply:
It is indeed on M1!

I've taken another stab at this on the latest main and am no longer seeing these issues. I thought I'd pulled the latest main before creating the branch for this PR, but maybe not?

Either way, filed https://github.com/kcctl/kcctl/pull/302 to add this missing test coverage and (of course 🙃) fix a bug that it unearthed.

Reply:
> It is indeed on M1!
>
> I've taken another stab at this on the latest main and am no longer seeing these issues.

Happy to hear! One thing we could do to further speed things up a bit is to use the _debezium-base_ image rather than _debezium_, as scanning the plug-in path definitely takes some time. That'd require to adjust a few tests though, e.g. on expected connector plug-ins.

Reply:
Logged https://github.com/kcctl/kcctl/issues/303 for this.